superset/db_engine_specs/bigquery.py

```diff
@@ -52,14 +52,14 @@
     from google.oauth2 import service_account
 
     dependencies_installed = True
-except ModuleNotFoundError:
```

Comment:
@drummerwolli out of interest why is `ModuleNotFoundError` not suffice? According to [this](https://stackoverflow.com/questions/62854761/python3-8-whats-the-difference-between-importerror-and-modulenotfounderror#:~:text=The%20ImportError%20is%20raised%20when,Python%203.6%20and%20newer%20versions.) StackOverflow post it states,

> TL;DR You should catch ModuleNotFoundError to detect when a package is not installed.